html-forms/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+    <p><label>Email:</label></p>
```

Comment:
This `label` element is not configured correctly to match with its `input`. Take a look at the documentation again, and pay close attention to mentions of the `for` and `id` attributes as related to label elements.